docs/running/binary.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: 'Running Conduit using a pre-built binary'
+sidebar_position: 0
+hide_title: true
+sidebar_label: "Pre-built Binary"
+---
+
+# Running Conduit using a pre-built binary.
+
+The recommended way of running Conduit on a local machine is using a pre-built binary. Below we'll walk through getting Conduit installed and running on your system.
+
+### Download
+First, download the [latest Conduit release](https://github.com/ConduitIO/conduit/releases/latest) for your platform.
+
+For an ARM based back Mac for example we'd grab the `conduit_0.8.0_Darwin_arm64.tar.gz` archive.
```

Comment:
Micro-nitpick: maybe update to 0.9.0 since we're about to release that. Or make it pull the version from somewhere.